app/models/message.rb

```diff
@@ -0,0 +1,11 @@
+class Message < ActiveRecord::Base
+
+  belongs_to :sender, class_name: 'User'
+  belongs_to :recipient, class_name: 'User'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.